introgression.md

```diff
@@ -90,21 +87,21 @@ def get_migrating_tracts(ts):
     migrating_tracts = []
     # Get all tracts that migrated into the neanderthal population
     for migration in ts.migrations():
-        if migration.dest == NEA:
+        if ts.population(migration.dest).metadata['name'] == 'Neanderthal':
             migrating_tracts.append((migration.left, migration.right))
     return np.array(migrating_tracts) 
 
 def get_coalescing_tracts(ts):
     coalescing_tracts = []
     tract_left = None
-    for tree in ts.trees():    
+    for tree in ts.trees():
         # 1 is the Eurasian sample and 2 is the Neanderthal
```

Comment:
I think it'd be better to stick with the integer population IDs here - it's quite a lot less efficient to decode the metadata etc, than to compare integers, and we're already hard-coding in 1 and 2 as the Eurasian IDs etc. You could do

```
neanderthal_id = demography["Neanderthal"].id
...
if mrca_pop == neaderthal_id
```

Reply:
I didn't know about `demography["Neanderthal"]`. Good call.

Reply:
Actually, we don't really have access to the demography object within those functions (or we shouldn't, anyway, unless we treat demography as global, or return it from `run_simulation`). So probably better to put

```
neanderthal_id = [p.id for p in ts.populations() if p.metadata['name']=='Neanderthal'][0]
```

at the top of the function?